src/metabase/query_processor/middleware/cumulative_aggregations.clj

```diff
@@ -24,6 +24,12 @@
     [:cum-count field] [:count field]
     [:cum-sum field]   [:sum field]))
 
+(defn- maybe-add
```

Comment:
You could just do `(fnil + 0 0)`

Reply:
I always forget fnil works for >1 arg. Thanks. Much nicer solution.